src/EllipsisNotation.jl

```diff
@@ -3,16 +3,45 @@ __precompile__()
 module EllipsisNotation
 
   import Base.getindex, Base.setindex!
-  const ..   = Val{:..}
+  const .. = Val{:..}()
+  const IndexType = Union{(VERSION < v"0.6-" ? Real : Int), Val{:..}}
```

Comment:
Oh yes, didn't catch that:

https://docs.julialang.org/en/stable/manual/interfaces/#abstract-arrays

Yeah, that should just be `Int` I think.